python_modules/dagster/dagster/core/execution/stats.py

```diff
@@ -208,7 +206,7 @@ class RunStepKeyStatsSnapshot(
     namedtuple(
         "_RunStepKeyStatsSnapshot",
         (
-            "run_id step_key status start_time end_time materializations expectation_results "
+            "run_id step_key status start_time end_time materialization_events expectation_results "
```

Comment:
I think this is okay (I don't think we store this anywhere), but it's very scary to modify a serializable object.  Can you confirm that this isn't stored?

Reply:
yes, this object is not stored